pkg/controller/pravega/pravega_controller.go

```diff
@@ -276,6 +277,20 @@ func configureControllerAuthSecrets(podSpec *corev1.PodSpec, p *api.PravegaClust
 	}
 }
 
+func configureControllerInfluxDBSecrets(podSpec *corev1.PodSpec, p *api.PravegaCluster) {
+	if p.Spec.Pravega.InfluxDBSecret != "" {
+		addSecretVolumeWithMount(podSpec, p, influxDBSecretVolumeName, p.Spec.Pravega.InfluxDBSecret,
```

Comment:
could  we consider accepting the mount path as a property from the user instead of hardcoding its value in a constant?